src/js/test/supervisors/FileManager.test.ts

```diff
@@ -68,6 +68,9 @@ const createStubs = (sandbox: SinonSandbox) => {
 };
 
 describe("FileManager", () => {
+  const active = "active";
```

Comment:
If this commit is necessary to achieve closer to 100% code coverage, can you explain that in the commit message? To a reviewer this commit looks random.

Reply:
No, this change is cosmetic. I'll remove it and make another PR with changes like this.

Reply:
This is a cosmetic change, I'll remove it and send another PR with that change.